### PR TITLE
Exclude scrooge core from bijection-scrooge, uniform declares  a higher ...

### PR DESCRIPTION
**Pull request by RayRoestenburg**
...scrooge core version dependency.

### CODE REVIEW

**Comment by RayRoestenburg** [comment]
@SamRoberts hmm looks like that # did have something to do with it. nice guess!

File: uniform-dependency/src/main/scala/au/com/cba/omnia/uniform/dependency/UniformDependencyPlugin.scala [hunk 1]
```diff
@@ -134,7 +134,7 @@ object UniformDependencyPlugin extends Plugin {
 
     def scrooge(scrooge: String = versions.scrooge, bijection: String = versions.bijection) = Seq(
       "com.twitter"              %% "scrooge-core"                  % scrooge,
-      "com.twitter"              %% "bijection-scrooge"             % bijection
+      "com.twitter"              %% "bijection-scrooge"             % bijection exclude("com.twitter", "scrooge-core_2.10")
```
Comment (stephanh) [inline uniform-dependency/src/main/scala/au/com/cba/omnia/uniform/dependency/UniformDependencyPlugin.scala]: Why aren't we just removing the `scrooge-core` import and just import `bijection-scrooge`?

**Comment by SamRoberts** [comment]
IIRC, the `scrooge-core` version is newer than the one used by `bijection-scrooge`. And we use the functionality in `scrooge-core` extensively, whereas to my knowledge, `bijection-scrooge` only gives us a few minor utility functions on top of that. If it was a choice between removing `scrooge-core` and using an older version, or removing `bijection-scrooge` and writing a few utility functions, I think I would prefer the later.

**Comment by RayRoestenburg** [comment]
+1 (although I don't have the full overview of usage you guys have)

I think there might be scala 2.11 versions for this but not sure if finagle
etc is ready yet and not sure what you guys are planning for scala upgrades.

**Pull request merged**